docs/data-scientists-root.md

```diff
@@ -1,49 +1,84 @@
+The support that investigation owners need from data scientists during experiment set-up includes:
 
-[This documentation](https://docs.google.com/document/d/155EUgzn22VTX8mFwesSROT3Z6JORSfb5VyoMoLra7ws/edit#) explains how to set up an experiment. The main support that Product Managers need from Data Scientists during experimentat set-up include: validating that the experimental design will answer their questions, sample size recommendations, writing custom metrics if needed, and explaining statistics.
+* validating that the experimental design will answer their questions
+* consulting on telemetry specifications
+* sample size recommendations
+* writing [custom metrics](jetstream/metrics.md) if needed
+* guidance about interpretation
 
 ## Sampling framework
 
-We cannot just go take a random sample of our clients. Instead, we must wait for a client to check for updates, and at that time, randomly assign the client to either a treatment or control. To handle this time discrepancy, each experiment starts with an enrollment period during which clients can enroll in the experiment. After the enrollment period ends, no more clients can be added to the experiment. See [this documentation](https://experimenter.info/jetstream/jetstream/#analysis-paradigm) for more information on how we handle clients who enroll at different times within the enrollment period, and how to control the time duration of an experiment.
+Experiment metrics are analyzed over a defined period of time since a client enrolls in an experiment. We report results for an analysis period (e.g. the first day after enrollment) after all clients have had a chance to experience the treatment for that duration. In order to return results in finite time, each experiment starts with an enrollment period of fixed length during which clients can enroll in the experiment. We modify the recipe to instruct clients to stop enrolling, and ignore clients that report enrollment anyway, after the declared enrollment period ends. The [Jetstream overview](jetstream/jetstream.md#analysis-paradigm) describes the analysis paradigm in more depth and how it relates to the length of an experiment.
 
 Because enrollment depends on checking for updates, samples will be skewed towards active users at the beginning of the enrollment period.
+We typically leave enrollment open for a week to account for weekly seasonality (e.g. weekday vs. weekend users) and to give clients who are active less often a chance to enroll.
+This makes our experiment population essentially a sample of WAU.
 
-For more nuances about sampling, enrollment and exposure (whether or not the client actually saw the treatment or control), see [this documentation](https://experimenter.info/client-sdk-states-and-lifecycle/#key-concepts).
+For more nuances about sampling, enrollment and exposure (whether or not the client actually saw the treatment or control), see [the experiment lifecycle overview](client-sdk-states-and-lifecycle.mdx).
```

Comment:
Not sure if this link will work. IIUC, the .mdx file will be converted to HTML which _might_ mean this link will then 404 once deployed. /cc @LZoog

Reply:
Docusaurus normalizes these; https://docusaurus.io/docs/docs-markdown-features#referencing-other-documents -- I had the same thought so I tested it locally.

Reply:
(Thanks for the close read and callout!)

Reply:
Wow, that's pretty awesome! `/giphy "the more you know"`